lib/generators/react_on_rails/templates/client/README.md

```diff
@@ -0,0 +1,97 @@
+Example NPM Package
+===========================
+We've included an example package.json from https://github.com/shakacode/react-webpack-rails-tutorial which should get you started with your React project.
+
+Starting the node.js server:
+```
+npm start
+```
+
+Building client javascript files for production:
+```
+npm run build:client
+```
+
+Building server javascript files for production:
+```
+npm run build:server
+```
+
+Building client javascript files for development:
+```
+npm run build:dev:client
+```
+
+Building server javascript files for development:
+```
+npm run build:dev:server
+```
+
+Running all linters:
+```
+npm run lint
+```
+
+Running eslint:
+```
+npm run eslint
+```
+
+Running jscs:
+```
+npm run jscs
+```
+
+dependencies vs devDependencies
+===========================
+Anything needed for heroku deployment needs to go in "dependencies", and anything not needed for heroku deployment should go in "devDependencies".
```

Comment:
this needs to be clarified in that anything needed for deployment source **OR** building needs to go in dependencies.

devDependencies includes linters and hot reloading stuff.